src/Game.h

```diff
@@ -99,6 +99,17 @@ class Game {
 	void SetTimeAccel(TimeAccel t);
 	void RequestTimeAccel(TimeAccel t, bool force = false);
 
+    /**
+     * Requests an increase in time acceleration
+     * @param force if set to false the system can reject the request under certain conditions
+     */
+    void RequestTimeAccelInc(bool force = false);
```

Comment:
See my other comment just now on C++ vs. C style comments.
(seems like we have C-style comments in other places, but at least not in this header file)

I think C-style comments is (only?) used at least when documenting code for Natrural docs LuaAPI that we use: http://eatenbyagrue.org/f/pioneer/codedoc/files/LuaBody-cpp.html